ARAnalytics.m

```diff
@@ -530,17 +530,16 @@ + (void)error:(NSError *)error withMessage:(NSString *)message
 
 + (void)pageView:(NSString *)pageTitle
 {
-    if (!pageTitle) return;
-    
-    [_sharedAnalytics iterateThroughProviders:^(ARAnalyticalProvider *provider) {
-        [provider didShowNewPageView:pageTitle];
-    }];
+    [self pageView:pageTitle withProperties:nil];
```

Comment:
That's a great point, and one that I admittedly didn't think of. A [quick search](https://github.com/orta/ARAnalytics/search?utf8=✓&q=pageView%3A) seems to show that no one is using it, but I'll do a more thorough search at the office.